sdk/storage/azure-storage-datamovement/src/tasks/download_blob_to_file_task.cpp

```diff
@@ -0,0 +1,88 @@
+#include "azure/storage/datamovement/tasks/download_blob_to_file_task.hpp"
+
+#include "azure/storage/datamovement/scheduler.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs { namespace _detail {
+  namespace {
+    constexpr uint64_t ChunkSize = 4 * 1024 * 1024;
+    static_assert(ChunkSize < static_cast<uint64_t>(std::numeric_limits<size_t>::max()), "");
+
+  } // namespace
+
+  // TODO: whether allow exuecute function throwing out exception? How to handle exception from each transfer step.
```

Comment:
I'll add support for error handling in separate PR. Let's just assume no error will happen during transfer and keep the TODO here for tracking.

Reply:
Updated TODO comments.